public/app/features/explore/LogsMetaRow.tsx

```diff
@@ -119,8 +168,25 @@ export const LogsMetaRow = React.memo(
                 };
               })}
             />
-            <ToolbarButton onClick={downloadLogs} variant="default" icon="download-alt">
-              Download logs
+
+            <ToolbarButton
```

Comment:
why not using [Context menu](https://developers.grafana.com/ui/latest/index.html?path=/docs/general-contextmenu--basic) or [Menu](https://developers.grafana.com/ui/latest/index.html?path=/story/general-menu--examples) instead of rolling out an adhoc component?

Reply:
Thank you @Elfo404! I changed it to use a `Dropdown` with `Menu` in 30d9de8a83! Really good suggestion.